docs/binary_input.md

```diff
@@ -13,7 +13,11 @@ shape: [N] with dtype: DT_STRING. Where N represents elements with binary data c
 
 ## Preparing for deployment
 Before processing in the target AI model, binary image data is encoded by OVMS to a NHWC layout in BGR color format.
-It will also be resized to the model or pipeline node resolution.
+It will also be resized to the model or pipeline node resolution. When the model shape is dynamic and image data shape
+is out of range it will be adjusted to the nearer border. For example, when model shape is: [1,3,100:200,200]:
```

Comment:
When using binary input, the layout of data is in NHWC order, meaning the 3 will be at last position.